app/src/pages/inside/filtersPage/filtersPage.jsx

```diff
@@ -202,11 +198,17 @@ export class FiltersPage extends Component {
             activeProject={activeProject}
             {...rest}
           />
-          {!filters.length && !loading && !this.props.filter && <NoFiltersBlock />}
-          {!filters.length && !loading && !!this.props.filter && this.getNoFiltersFound()}
-          {filters &&
-            !!filters.length &&
-            !!pageCount &&
+          {!filters.length &&
+            !loading &&
+            (this.props.filter ? (
```

Comment:
Nested ternary expressions are difficult to read and understand. I suggest splitting this into render functions.

Reply:
FIxed